data_analysis/static_gtfs_analysis.ipynb

```diff
@@ -774,27 +732,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 57,
+   "execution_count": 93,
    "id": "d812de6d",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# now group to get trips by hour by date by route\n",
-    "route_daily_hourly_summary = trip_summary.groupby(by = ['raw_date', 'route_id', 'arrival_hour'])['trip_id'].count().reset_index()"
+    "def summarize_and_save(trip_summary): \n",
+    "    # now group to get trips by hour by date by route\n",
+    "    route_daily_hourly_summary = trip_summary.groupby(by = ['raw_date', 'route_id', 'arrival_hour'])['trip_id'].count().reset_index()\n",
+    "\n",
+    "    route_daily_hourly_summary.rename(columns = {'arrival_hour': 'hour', 'trip_id': 'trip_count', 'raw_date': 'date'}, inplace = True)\n",
+    "    route_daily_hourly_summary.date = route_daily_hourly_summary.date.dt.date\n",
+    "    route_daily_hourly_summary.to_csv(f's3://chn-ghost-buses-private/schedule_summaries/route_level/schedule_route_daily_hourly_summary_v{VERSION_ID}.csv', index = False)\n",
```

Comment:
It would probably be good to set a variable such as `bucket_type = 'public'` at the beginning of the notebook so that the `s3` paths can be written as `f'3://chn-ghost-buses-{bucket_type}/...`. It would also be good to comment out the `to_csv` method because it will throw errors.